tests/admin_views/tests.py

```diff
@@ -864,6 +864,27 @@ def test_adminsite_display_site_url(self):
         self.assertEqual(response.context['site_url'], '/my-site-url/')
         self.assertContains(response, '<a href="/my-site-url/">View site</a>')
 
+    def test_submit_row_templatetag(self):
+        """
+        #13875 - submit_row templatetag should pass whole context
```

Comment:
ticket reference isn't needed -- we try to reserve that for obscure issues that can't be easily described.
"template tag" (two words)
add period